servers/rendering/renderer_rd/effects/post_processor.cpp

```diff
@@ -1,5 +1,5 @@
 /**************************************************************************/
-/*  tone_mapper.cpp                                                       */
+/*  post_processor.cpp                                                    */
```

Comment:
I would prefer not to rename the tone_mapper. While the new name is more accurate, it creates a lot of noise in the git history for no tangible benefit

Reply:
The main reason is that it took me a while to understand that the tonemapper was doing most of the work before serving the image. There is so much more than tonemapping in the GLSL shader but i can revert it. (I may restart from scratch again because reverting that will be a pain)